tests/test_jx/test_deep_ops.py

```diff
@@ -820,9 +820,7 @@ def test_aggs_on_parent_and_child(self):
                     {"o": 1, "v": NULL, "s": 0},
                     {"o": 2, "v": "b", "s": 1},
                     {"o": 2, "v": "c", "s": 0},
-                    {"o": 2, "v": NULL, "s": 1},
-                    {"o": NULL, "v": "b", "s": 0},
-                    {"o": NULL, "v": "c", "s": 0}
+                    {"o": 2, "v": NULL, "s": 1}
```

Comment:
Thinking a bit more, this might be a problem:  The cube building logic demands the resultset be in order to easily fill the cube of values.  

    r2c = index_to_coordinate(dims)  # WORKS BECAUSE THE DATABASE SORTED THE EDGES TO CONFORM

Consider the same test, with no `v` missing:

            "data": [
                {"o": 1, "a": {"_a": [
                    {"v": "b", "s": False},
                    {"v": "c"}
                ]}},
                {"o": 1, "a": {"_a": {
                    "v": "b",
                    "s": False
                }}},
                {"o": 2, "a": {"_a": [
                    {"v": "b", "s": True},
                ]}}
                # REMOVE {"o": 2, "a": {"_a": {"s": False}}}
            ]

Your logic will demand that rows with missing `v` need not be in the list, or the table; but you must return them to fill the cube properly with the existing logic.

I believe it is easier to add logic that will return the extra `null` slices, rather than mess with the python logic that interprets the result.